README.md

```diff
@@ -10,19 +10,18 @@ React Router keeps your UI in sync with the URL. It has a simple API with powerf
 ### Docs & Help
 
 - [Tutorial – do this first!](https://github.com/reactjs/react-router-tutorial)
-- [Guides and API docs](https://github.com/reactjs/react-router/tree/master/docs)
+- [Guides and API docs](/docs)
```

Comment:
Depending on how we do the branching for 3.0, we might need to make this a fully-qualified URL again to link to the right branch for docs.